src/java.desktop/share/classes/javax/swing/text/html/CSS.java

```diff
@@ -1569,12 +1570,12 @@ static int getIndexOfSize(float pt, StyleSheet ss) {
                 current++;
             }
             if (last != current) {
-                temp.addElement(value.substring(last, current));
+                temp.add(value.substring(last, current));
             }
             current++;
         }
         String[] retValue = new String[temp.size()];
-        temp.copyInto(retValue);
+        temp.toArray(retValue);
```

Comment:
Why not `return temp.toArray(new String[0]);` like in other places in the last commit?

Reply:
Good catch! Update here too.